MONITORING_LOGGING.md

```diff
@@ -1 +1,103 @@
-Write based on: https://docs.google.com/document/d/1gbc-e9Nf51Tzz3rGbLgbQ8jPkr9KozFcVYnYLeCC1Xs/edit#heading=h.b88jdrn065o9
\ No newline at end of file
+# Logging and Monitoring
```

Comment:
So one of the things we are doing on Auth0 is opportunistic logging where we save details until there is an issue and then send the logs to splunk.

Reply:
Not sure if we want to add that as a mechanism ?

Reply:
@sandeepasrani added it to the Logging Libraries & Aggregation section, let me know what you think.